wordpress-mobile/WordPress-iOS
# Adds some site creation UI improvements

Fixes #11686 subtask: Update the `SiteAssemblyContentView` UI

**Changes:**
- `AppImages.xcassets`
  - Added a new imageset `site-creation-loading` that is displayed while the user is waiting for their site to be created in both dark and light modes

- `SiteAssemblyContentView`:
  - Added 3 new properties:
    - `statusTitleLabel` and `statusSubtitleLabel`: Display the title and subtitle copy specified here: https://github.com/wordpress-mobile/WordPress-iOS/issues/11686#issuecomment-558860091
    - `statusImageView`: Displays the new `site-creation-loading` image

**To test:**
- Launch the app
- Go to the 'My Sites' tab
- Go to the 'Switch Sites' view
- Tap the + button to add a new site
- Tap the `Create WordPress.com site` option
- Tap any site option
- Tap Skip to skip the about
- Tap Skip to skip the basic information
- Search for any domain and select it
- Tap the 'Create Site' button

Example expected screenshots

| Device  | iOS Version | Mode | Screen |
|----------|----------|----------|----------|
| iPhone 5s  |  11.4 | N/A - Light |<img src="https://user-images.githubusercontent.com/793774/70591621-986e7000-1b8b-11ea-8ae0-6335d8820a4b.png" width="50%" align="middle"/> |
| iPad Air 2 | 12.4 | N/A - Light |<img src="https://user-images.githubusercontent.com/793774/70592108-783fb080-1b8d-11ea-963f-6b81ae5e88ad.png" width="50%"/>  |
| iPhone 11 Pro | 13.2.2 | Light |<img src="https://user-images.githubusercontent.com/793774/70591962-d7e98c00-1b8c-11ea-9f06-4f3d697dd278.png" width="50%"/>  |
| iPhone 11 Pro | 13.2.2 | Dark |<img src="https://user-images.githubusercontent.com/793774/70593239-1c772680-1b91-11ea-9e85-53e5aa7a1dbf.png" width="50%"/>  |
| iPad Pro (12.9 inch) | 13.2.2 | Light |<img src="https://user-images.githubusercontent.com/793774/70592449-b2f61880-1b8e-11ea-91ba-ad120994ede5.png" width="50%"/>  |
| iPad Pro (12.9 inch) | 13.2.2 | Dark |<img src="https://user-images.githubusercontent.com/793774/70592589-344dab00-1b8f-11ea-906d-030127c42dbb.png" width="50%"/>  |

**Other notes:**
- I noticed there was the `TitleSubtitleHeader` class that already handles the title/subtitle flow which I initially tried to use, but in practice the constraints that were being added to it internally didn't match the design I was given so I reverted back to my own title/subtitle properties

**PR submission checklist:**
